src/1.18/chunk.js

```diff
@@ -28,12 +28,15 @@ module.exports = (ChunkColumn, registry) => {
 
         const biomePalette = biomeSection.data.palette ? biomeSection.data.palette : [biomeSection.data.value] // "SingleValue" palette mess
 
-        const bitsPerBlock = Math.ceil(Math.log2(blockPalette.length))
+        let bitsPerBlock = Math.ceil(Math.log2(blockPalette.length))
         const bitsPerBiome = Math.ceil(Math.log2(biomePalette.length))
 
         if (!bitsPerBlock) {
           blockStates = nbt.comp({ palette: nbt.list(nbt.comp(blockPalette)) })
         } else {
+          if (bitsPerBlock === 1 || bitsPerBlock === 2 || bitsPerBlock === 3) {
+            bitsPerBlock = 4
+          }
```

Comment:
@extremeheat I copied it from here https://github.com/PrismarineJS/prismarine-provider-anvil/blob/422aed5db94f341c3807f1a918c7b83c9ebcfe20/src/1.18/chunk.js#L130 I thought it cant be less than 4 (definitely had issues with saving).

but since I didn't add tests I will move it to a separate pr